src/community/galaxy-admins/future/index.md

```diff
@@ -10,7 +10,7 @@ title: GalaxyAdmins Future Directions
 
 
 
-First, some background history, and then a list of open questions.
+First, some background history, and then a list of open questions cira 2015. The regular online [meetings](/src/community/galaxy-admins/meetups/index.md) resumed and continued until early 2017, with a final in person meeting during  [GCCBOSC 2018](https://gccbosc2018.sched.com/event/FMQW/bof-galaxyadmins). Discussion has since shifted to [Galaxy Project Gitter "Admins"](https://gitter.im/galaxyproject/admins).
```

Comment:
```suggestion
First, some background history, and then a list of open questions circa 2015. The regular online [meetings](/src/community/galaxy-admins/meetups/index.md) resumed and continued until early 2017, with a final in person meeting during  [GCCBOSC 2018](https://gccbosc2018.sched.com/event/FMQW/bof-galaxyadmins). Discussion has since shifted to [Galaxy Project Gitter "Admins"](https://gitter.im/galaxyproject/admins).
```